README.md

```diff
@@ -7,8 +7,8 @@ The original SAS implementation can be found [here](https://www.nber.org/data/cm
 Currently, hccpy supports CMS-HCC V22, V23, V24, ESRD, and HHS-HCC Year 2019.
```

Comment:
It also covers HHS-HCC Year 2021 (needs to be added)

Reply:
I've added this and tidied up the text a bit. Let me know what you think.